### PR TITLE
docs: Tina > TinaProvider

### PR DESCRIPTION
**Pull request by DirtyF**
Reflect changes from https://github.com/tinacms/tinacms/pull/873

### CODE REVIEW

File: content/docs/cms.md [hunk 1]
```diff
@@ -24,31 +23,31 @@ const cms = new TinaCMS()
 
 The `TinaCMS` constructor receives an object that can be used to configure CMS behavior. See [CMS Configuration](#cms-configuration) for details.
 
-### The &lt;Tina&gt; Component
```
Comment (dwalkr) [inline content/docs/cms.md]: This renders as shown:

image

File: content/docs/cms.md [hunk 2]
```diff
@@ -24,31 +23,31 @@ const cms = new TinaCMS()
...
+### The <TinaProvider> Component
```
Comment (dwalkr) [inline content/docs/cms.md]: this renders as shown:

image

Comment (dwalkr) [reply content/docs/cms.md]:
```suggestion
### The &lt;TinaProvider&gt; Component
```

Comment (spbyrne) [reply content/docs/cms.md]: Is this an API change (hence the docs changes)?

**Pull request merged**